Components/Card/card.css

```diff
@@ -0,0 +1,79 @@
+@import url(/css/Allcss.css);
+* {
+    margin: 0;
+    padding: 2rem;
+    box-sizing: border-box;
+    color: #eee;
+  }
+  
+  .card {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 20rem;
+    border: 2px solid var(--border-color);
+  }
+  .card-badge {
+    position: absolute;
+    background-color: var(--primary-color);
+    color: #eee;
+    padding: 1px;
+    top: 5px;
+    left: 0px;
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
```

Comment:
You can use rem here instead of px

Reply:
Better not to use rem for less than 10px